api/app/schemas/morecast_v2.py

```diff
@@ -73,7 +73,7 @@ class MoreCastForecastInput(BaseModel):
     precip: float
     wind_speed: float
     wind_direction: int | None = None
-    grass_curing: float | None = None
+    grass_curing: int | None = None
```

Comment:
They don't actually, they come from the WF1 as floats (always ##.0). Not sure if changing this to an int is bad practice or not...

Reply:
I think we should keep them floats if that's how they come from WF1, otherwise we'll get validation errors and we won't get grass curing

Reply:
![image](https://github.com/bcgov/wps/assets/42357257/c46189dd-90d0-47aa-a73a-f376b3186655)

I was going to change it back to a float but just wanted to do a sanity check. Looking into it a bit, it also seems like `windDirection` comes from WF1 as a float, but use int in our schema. I haven't seen any validation errors, maybe pydantic is doing some automatic conversions behind the scenes? Unless thunder client/vscode is lying to me, I've checked these values a couple ways

Reply:
Yeah wind direction should always be an integer, as defined by forecasters. Since we don't know if grass curing can be a fractional percentage, the safest way to make sure we always accept a value is to treat it as a float. Unless it's been defined by forecasters as an integer?

Reply:
No it hasn't been defined to my knowledge, I'll go the safe route